packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -172,7 +172,7 @@ abstract class InteractiveInkFeatureFactory {
     BorderRadius? borderRadius,
     ShapeBorder? customBorder,
     double? radius,
-    required VoidCallback? onRemoved,
```

Comment:
@a14n Can you confirm that there was no reason for adding the `required` here in the first place?